CHANGELOG.md

```diff
@@ -6,8 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- Tensorflow hub NLP models.
 - Notes on development dependencies in `setup.py` files to codebase notes.
-
```

Comment:
We want to maintain this blank line (all releases have blank lines between their last `- list item` and the previous release header.